lib/streams/documentStream.js

```diff
@@ -28,7 +28,22 @@ function createDocumentStream(id_prefix, stats) {
         if (record.POSTCODE) {
           addrDoc.setAddress( 'zip', record.POSTCODE );
         }
-
+        
+        if (useOaAdminFields) {
+          if (record.CITY) {
+            addrDoc.parent.locality.push(record.CITY);
```

Comment:
Hi @trescube 

I've finally had a bit of time to get back to this. it looks like the [addParent function](https://github.com/pelias/model/blob/5463126698feb1ac63ef0b6b257d4966dc6f928f/Document.js#L149-L157) takes 4 arguments, my current approach provides the first two (being a parent type and a value) however it looks like I also need to supply a third argument for parent ID which I don't really have. Is there any easy way around this or would I be breaking the data models?

Cheers
Rowan